e2e/atlas_alertconfig_test.go

```diff
@@ -36,6 +47,61 @@ func TestAtlasAlertConfig(t *testing.T) {
 	atlasEntity := "atlas"
 	alertConfigEntity := "alert-config"
 
+	t.Run("Create", func(t *testing.T) {
```

Comment:
Let's not do e2e until we can delete it or will be creating a bunch of useless alerts